### PR TITLE
Write out EVI layer for modis indices and also fix a bunch of other stuff

### PR DESCRIPTION
**Pull request by ags-tolson**
* Write out EVI layer for modis indices
* Migrate entirely to modis collection 6 (except where we can't)
* Replace ftp & http with https for asset URLs, and send auth creds for listing URLs as well as content URLs
* increase timeout for fetchces
* QC layer for indices product (i don't understand how the computations work though)
* make `remove_files` quit being dumb when removing nonexistent files
* Fixes for various busted tests

Tested (manually) with `gips_process modis -d2009-001 -s gips/test/NHseacoast.shp -v4`.

### CODE REVIEW

**Comment by ags-tolson** [comment]
I mentioned this in slack but this PR is ready to be reviewed.

**Comment by bhbraswell** [comment]
Besides the migration from C5 to C6, probably the most significant change is the fix to indices: previously the EVI layer was not being written out. @ags-tolson thanks for taking this on.

File: gips/data/modis/modis.py [hunk 1]
```diff
@@ -265,7 +267,7 @@ class modisData(Data):
     _productgroups = {
         "Nadir BRDF-Adjusted 16-day": ['indices', 'quality'],
         "Terra/Aqua Daily": ['snow', 'temp', 'obstime', 'fsnow'],
-        # "Terra 8-day": ['ndvi8', 'temp8tn', 'temp8td'], # ndvi8 is deactivated for now
+        "Terra 8-day": ['ndvi8', 'temp8tn', 'temp8td'], # ndvi8 is deactivated for now
```
Comment (bhbraswell) [inline gips/data/modis/modis.py]: It looks like there are two keys with the same name, is that right?

Comment (ags-tolson) [inline gips/data/modis/modis.py]: You're right, I'll fix that.

**Comment by bhbraswell** [comment]
based on user need, and my quick review, can this get merged and brought out into the world?

**Comment by ircwaves** [comment]
yes

**Comment by bhbraswell** [comment]
wait, is this the one with the re-animated ndvi8? that's the one they need.

File: gips/data/modis/modis.py [hunk 2]
```diff
@@ -265,8 +267,7 @@ class modisData(Data):
     _productgroups = {
         "Nadir BRDF-Adjusted 16-day": ['indices', 'quality'],
         "Terra/Aqua Daily": ['snow', 'temp', 'obstime', 'fsnow'],
-        # "Terra 8-day": ['ndvi8', 'temp8tn', 'temp8td'], # ndvi8 is deactivated for now
-        "Terra 8-day": ['temp8tn', 'temp8td'],
+        "Terra 8-day": ['ndvi8', 'temp8tn', 'temp8td'],
```
Comment (ircwaves) [inline gips/data/modis/modis.py]: yes

**Pull request merged**